targets/TARGET_STM/TARGET_STM32H7/TARGET_STM32H743xI/TARGET_NUCLEO_H743ZI2/system_clock.c

```diff
@@ -72,7 +72,7 @@ void SetSysClock(void)
 #if ((CLOCK_SOURCE) & USE_PLL_HSE_XTAL)
         /* 2- If fail try to start with HSE and external xtal */
         if (SetSysClock_PLL_HSE(0) == 0)
-#endif
+#endif 
```

Comment:
minor comment, remove space

Reply:
OK, space removed = )
Also I changed error message to compile time check. Check it please, @jeromecoutant